verticapy/plotting/_highcharts/acf.py

```diff
@@ -94,6 +94,9 @@ def draw(
         """
         kind = "PACF" if self.layout["pacf"] else "ACF"
         chart, style_kwargs = self._get_chart(chart, style_kwargs=style_kwargs)
+        if "colors" in style_kwargs:
+            self.init_style_confidence["fillColor"] = style_kwargs["colors"]
```

Comment:
what if style_kwargs["colors"] is not a list or the opposite?

Reply:
have updated